app/models/comment.rb

```diff
@@ -3,8 +3,10 @@ class Comment < ActiveRecord::Base
   belongs_to :owner, class_name: 'User', foreign_key: 'owner_user_id'
   belongs_to :language
 
-  validates_presence_of :owner_user_id, :commentable_id, :commentable_type, :body, :language
+  validates_presence_of :owner_user_id, :commentable_id, :commentable_type,
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.